.goreleaser.yml

```diff
@@ -97,6 +97,13 @@ archives:
     format_overrides:
     - goos: windows
       format: zip
+    wrap_in_directory: "true"
+    files:
+      - docs/*
+      - LICENSE.md
```

Comment:
File is currently LICENSE, so when I tested with "make test-release" it failed due to looking for LICENSE.md
This line should be changed to "LICENSE".

Reply:
Could've sworn I included that fix; in any case, just pushed an updated branch.